ext/spl/spl_iterators.c

```diff
@@ -1521,6 +1545,8 @@ static spl_dual_it_object* spl_dual_it_construct(INTERNAL_FUNCTION_PARAMETERS, z
 			cfi->object = cfi->fcc.object;
 			if (cfi->object) GC_ADDREF(cfi->object);
 			intern->u.cbfilter = cfi;
+
+			ce = get_ce_or_aggregate_ce(&zobject, &retval, &inc_refcount);
```

Comment:
This `ce` does not appear to be used. We should also be checking the failure condition here.